Makefile

```diff
@@ -58,7 +58,7 @@ run-server:
 
 .PHONY: run-celery
 run-celery:
-	poetry run celery -A node.config.celery worker --loglevel=INFO
+	poetry run celery -A node.config.celery worker --loglevel=INFO  --pool=eventlet --concurrency=30
```

Comment:
@olegtropinin why eventlet?

Reply:
@olegtropinin let's go with prefork or process and autoscale for now, we will optimize with async approach later if needed

Reply:
I found information that "eventlet" pool type have greater concurrency than prefork. Also it's good for API calls. It was a case why I selected prefork.
I removed "eventlet" as you said. It has "prefork" default pool now.

You are right no need to have too many threads for development. I removed concurrency argument from Makefile (make run-celery).